core/vector.h

```diff
@@ -92,6 +92,15 @@ class Vector {
 
 	void append_array(Vector<T> p_other);
 
+	bool has(const T &p_val) {
+		for (int i = 0; i < size(); i++) {
+			if (get(i) == p_val) {
+				return true;
+			}
+		}
+		return false;
+	}
```

Comment:
`Array::has()` relies on `find()`, which is also implemented for `Vector`/`CowData`. Maybe it would be faster than this? (Didn't try.)

Reply:
I'm not sure which is faster, but they both look similar, and it would be simpler and better for consistency with Array to use find, so I changed it to find.